File: app/src/main/java/com/qingshuige/tangyuan/viewmodels/PostHeaderAdapter.java
package com.qingshuige.tangyuan.viewmodels;

import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

public class PostHeaderAdapter extends RecyclerView.Adapter<PostHeaderAdapter.ViewHolder> {
    @NonNull
    @Override
    public PostHeaderAdapter.ViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
        return null;
    }

    @Override
    public void onBindViewHolder(@NonNull PostHeaderAdapter.ViewHolder holder, int position) {

    }

    @Override
    public int getItemCount() {
        return 0;
    }

    static class ViewHolder extends RecyclerView.ViewHolder {
        public ViewHolder(@NonNull View itemView) {
            super(itemView);
        }
    }
}

File: app/src/main/java/com/qingshuige/tangyuan/viewmodels/PostHeaderAdapter.kt
package com.qingshuige.tangyuan.viewmodels

import android.view.View
import android.view.ViewGroup
import androidx.recyclerview.widget.RecyclerView

class PostHeaderAdapter : RecyclerView.Adapter<PostHeaderAdapter.ViewHolder>() {

    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
        // TODO: 实现具体的布局
        return ViewHolder(View(parent.context))
    }

    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
        // TODO: 实现数据绑定
    }

    override fun getItemCount(): Int = 0

    class ViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView)
}